src/Microsoft.AspNetCore.Mvc.DataAnnotations/Internal/MvcDataAnnotationsLocalizationOptionsSetup.cs

```diff
@@ -9,14 +9,13 @@ namespace Microsoft.AspNetCore.Mvc.DataAnnotations.Internal
     /// <summary>
     /// Sets up default options for <see cref="MvcDataAnnotationsLocalizationOptions"/>.
     /// </summary>
-    public class MvcDataAnnotationsLocalizationOptionsSetup : ConfigureOptions<MvcDataAnnotationsLocalizationOptions>
+    public class MvcDataAnnotationsLocalizationOptionsSetup : IConfigureOptions<MvcDataAnnotationsLocalizationOptions>
     {
-        public MvcDataAnnotationsLocalizationOptionsSetup()
-            : base(ConfigureMvc)
-        {
-        }
-
-        public static void ConfigureMvc(MvcDataAnnotationsLocalizationOptions options)
+        /// <summary>
```

Comment:
`/// <inheritdoc />` should suffice. You could add this as a remark to the type if you'd like, but that doesn't seem particularly necessary.